src/SchemaGenerator.ts

```diff
@@ -20,7 +20,7 @@ export class SchemaGenerator {
         protected readonly config?: Config,
     ) {}
 
-    public createSchema(fullName?: string): Schema {
```

Comment:
If we only accept arrays, the api would be simpler.

Reply:
I changed this back to `string | string[]`. Many of the tests pass `"*"` or `MyObject`, and if we change to `"*"` or `["MyObject"]` it would mean a lot bigger diff. Happy to do that if you think that's better. It seemed like keeping this and then just doing a `castArray()` was easier, but happy to take suggestions of course.

Reply:
I'd vote to keep `"*"` as a valid value just to make this a non-breaking change.

Reply:
At the CLI level or the API level? At the CLI level, I agree. At the API level I am okay with making a breaking change if it significantly simplifies code otherwise.